internal/node/node_launcher.sh

```diff
@@ -114,7 +114,24 @@ TEMPLATED_env_vars
 # This redirects to stderr so it doesn't interfere with Bazel's worker protocol
 # find . -name thingImLookingFor 1>&2
 
-readonly node=$(rlocation "TEMPLATED_node")
+# Check environment
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=linux;;
+    Darwin*)    machine=darwin;;
+    CYGWIN*)    machine=windows;;
+    MINGW*)     machine=windows;;
+    MSYS_NT*)   machine=windows;;
+    *)          machine=linux;;
+esac
+
+case "${machine}" in
+  linux) readonly node=$(rlocation "TEMPLATED_node_linux");;
+  darwin) readonly node=$(rlocation "TEMPLATED_node_darwin");;
+  windows) readonly node=$(rlocation "TEMPLATED_node_windows");;
+  *) readonly node=$(rlocation "TEMPLATED_node_linux");;
```

Comment:
should log something noisy in this case, maybe even tell user to file an issue and include the uname?

Reply:
added something noisy and error handling if the chosen node executable is not in runfiles